lib/common/visualmetricsWrapper.py

```diff
@@ -49,12 +50,43 @@ def colors_are_similar(a, b, threshold=30):
     return similar
 
 
-def find_tab_view(input_file, viewport):
+def find_terminal_view(input_file, viewport):
+    """
+    Find the Region of imageUtil.CropRegion.TERMINAL.
+    @param input_file: image file.
+    @param viewport: {x, y', width, height} of VIEWPORT.
+    @return: {x, y', width, height}
     """
+    try:
+        im = Image.open(input_file)
+        im_width, im_height = im.size
 
-    @param file:
-    @param viewport:
-    @return:
+        base_x = int(viewport.get('x', 0))
+        base_y = int(viewport.get('y', 0))
+        base_width = int(viewport.get('width', im_width))
+        base_height = int(viewport.get('height', 0))
+
+        # get Terminal location
+        terminal_location = get_terminal_location(base_x, base_y, base_width, base_height)
+
+        # Adjust the height (do not over the image height)
+        terminal_y = terminal_location.get('y')
+        bottom = min(im_height, terminal_y + terminal_location.get('height') + 25)
```

Comment:
This shift offset is difference between different base, `browser` at runtime and `viewport` at screenshot.